src/main/java/pl/ateam/disasteralerts/message/ToastMessageFacade.java

```diff
@@ -0,0 +1,11 @@
+package pl.ateam.disasteralerts.message;
+
+import org.springframework.stereotype.Component;
+import pl.ateam.disasteralerts.message.dto.ToastMessageDTO;
+
+@Component
+public interface ToastMessageFacade {
+
+    ToastMessageDTO buildMessage(ToastMessageType type, String header, String message);
+    ToastMessageDTO buildMessage(ToastMessageType type, String icon, String header, String message);
```

Comment:
Super, fajnie że jest też zaktualizowany toast do edycji danych.


Zgaduję że to jest propozycja implementacji każdej fasady jako interfejsu zamiast klasy o której wspominałeś? :) Mi pasuje takie rozwiązanie, nawet w jednym z wcześniejszych projektów robiłem podchody żeby osiągnąć podobny efekt ale moje podejście było złe bo nie rozumiałem jak poukładać sobie modyfikatory dostępu